explainers/hidden-content-explainer.md

```diff
@@ -200,24 +205,87 @@ representing the active match is collapsed or it has a style which makes it
 invisible, the scroll will be canceled. Otherwise, the target match will be
 scrolled into view.
 
-### Alternatives Considered
+## Accessibility
+
+`content-visibility: hidden-matchable` subtrees will not be included in the
+accessibility tree because they are not visible on the screen. This is the same
+as `content-visibility: hidden` and offscreen/unrendered `content-visibility:
+auto` subtrees.
```

Comment:
I would omit mentioning content-visibility: auto since we're in process of making them included in the AX tree

Reply:
thanks, done